tools/contactsEmailFilter.js

```diff
@@ -25,6 +34,13 @@ const verifyMails = async () => {
 	});
 
 	bec.on('data', data => {
+		if (
```

Comment:
This is the wrong approach. What you are doing:

1. Take email from list
2. Check with SMTP server if it is valid
3. Check if it is blacklisted and set "invalid"
4. See the SMTP server result and set "valid", "invalid" or "unsure".

What should be done:

1. Take email from list
2. Check if it is blacklisted and set "invalid"
3. For others, check with SMTP server and set "valid", "invalid" or "unsure".

This prevents wasteful SMTP checks for blacklisted emails.